### PR TITLE
Fix #2038: File formatting under numericinput package

### LINKED ISSUE #2038: Fix file formatting under numericinput package
**Describe the bug**
All the files, under the package `domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput` are not in the correct format. 

**Expected behavior**
File format should be follow from this file - [TextInputEqualsRuleClassifierProviderTest](https://github.com/oppia/oppia-android/blob/develop/domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt
)

**Correct Order**
Constant values
Dagger injection
Local variables
Test Before setup
Tests
Dagger Setup

### PR DESCRIPTION
**Pull request by Victor-Titan**
## Explanation

Fixes #2038 Formatted the files under the numericinput package as specified in issue #2038

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

PTAL @anandwana001

### CODE REVIEW

**Comment by Victor-Titan** [comment]
@anandwana001 I can't understand the error it's showing in the checks. Can you please help?

File: domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsGreaterThanRuleClassifierProviderTest.kt [hunk 1]
```diff
@@ -22,16 +22,24 @@ import javax.inject.Singleton
...
+    InteractionObjectTestBuilder.createString(value = "test")
+  private val POSITIVE_INT_VALUE_3 =
+    InteractionObjectTestBuilder.createInt(value = 3)
+  private val POSITIVE_INT_VALUE_1 =
+    InteractionObjectTestBuilder.createInt(value = 1)
+  private val NEGATIVE_INT_VALUE_1 =
+    InteractionObjectTestBuilder.createInt(value = -1)
+  private val NEGATIVE_INT_VALUE_3 =
+    InteractionObjectTestBuilder.createInt(value = -3)
   @Inject
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsGreaterThanRuleClassifierProviderTest.kt:44]: leave an empty line above

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @Victor-Titan 
one change else LGTM

**Comment by Victor-Titan** [comment]
> Thanks @Victor-Titan
> one change else LGTM

@anandwana001 Please check now, I've made the changes.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @Victor-Titan

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**